gui-apps/gtkgreet/gtkgreet-0.6.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="gtk based greeter for greetd"
+HOMEPAGE="https://git.sr.ht/~kennylevinsen/gtkgreet"
+
+inherit meson
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.sr.ht/~kennylevinsen/gtkgreet.git"
+else
+	SRC_URI="https://git.sr.ht/~kennylevinsen/gtkgreet/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+layershell +man"
+
+DEPEND="
+	dev-libs/json-c
```

Comment:
missing subslot op `:=`

you can see if you need subslot deps by running `eshowkw json-c`
you see, it has 3 subslots currently. if a new one introduced or user installs 0/5 instead of 0/4 you'd likely need a rebuild.

```

Keywords for dev-libs/json-c:
           |                             |   u     |  
           | a   a     p s     a   r     |   n     |  
           | m   r h   p p   s l i i m m | e u s   | r
           | d a m p p c a x 3 p a s 6 i | a s l   | e
           | 6 r 6 p p 6 r 8 9 h 6 c 8 p | p e o   | p
           | 4 m 4 a c 4 c 6 0 a 4 v k s | i d t   | o
-----------+-----------------------------+---------+-------
   0.12    | + + + + + + + + + ~ ~ o o ~ | 5 o 0/2 | gentoo
-----------+-----------------------------+---------+-------
 0.13.1-r1 | + + + + + + + + + ~ ~ ~ o ~ | 6 o 0/4 | gentoo
-----------+-----------------------------+---------+-------
[I]0.14-r3 | + + + + + + + + + ~ ~ o o ~ | 7 o 0/5 | gentoo
   9999    | o o o o o o o o o o o o o o | 7 o     | gentoo
```

Reply:
Like I had a cursory glance in the slots, forgot to check subslots :crying_cat_face: 
Would be nice if `pkgcheck scan` could tell this !

In any case, I've fixed it.